services/src/main/java/org/keycloak/services/clientpolicy/context/PushedAuthorizationRequestContext.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.services.clientpolicy.context;
```

Comment:
Similarly like for other PR related to CIBA, my vote is to make sure that advanced preview features like CIBA or PAR are as isolated as possible from the other codebase and fully wrapped within it's own package structure. Hence the package like `org.keycloak.protocol.oidc.par.executor` might be better.

Reply:
@mposolda I see, how about the following packages?

- `org.keycloak.protocol.oidc.par.executor` : PAR's executors are located.
- `org.keycloak.protocol.oidc.par.condition` : PAR's conditions are located.
- `org.keycloak.protocol.oidc.par.context` : PAR's contexts are located.

Reply:
> @mposolda I see, how about the following packages?
> 
>     * `org.keycloak.protocol.oidc.par.clientpolicy.executor` : PAR's executors are located.
> 
>     * `org.keycloak.protocol.oidc.par.clientpolicy.condition` : PAR's conditions are located.
> 
>     * `org.keycloak.protocol.oidc.par.clientpolicy.context` : PAR's contexts are located.

+1